libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -74,6 +77,13 @@ export class SkyAgGridWrapperComponent
     this.#isCompact.next(value);
   }
 
+  /**
+   * The minimum height of the grid in pixels. The default value is `50`.
+   */
+  public readonly minHeight = input<number, unknown>(50, {
```

Comment:
should we have a unit test covering this?

Reply:
Good call. Added.